File: abc/abc362/abc362_a/abc362_a.cs
using System.Collections.Generic;
using static System.Console;

public class A
{
  static void Main()
  {
    List<int> rgb = ReadLine().Split().Select(int.Parse).ToList();
    string c = ReadLine();

    int idx = c switch
    {
      "Red" => 0,
      "Green" => 1,
      _ => 2,
    };

    rgb.RemoveAt(idx);
    WriteLine(rgb.Min());
  }
}

File: abc/abc362/abc362_a/abc362_a.rs
use proconio::input;
use std::cmp;

fn main() {
    input! {
      r: u8,
      g: u8,
      b: u8,
      c: String,
    }

    println!("{}", match c.as_str() {
      "Red" => cmp::min(g, b),
      "Green" => cmp::min(b, r),
      _ => cmp::min(r, g),
    });
}